pyproject.toml

```
@@ -5,3 +5,541 @@ requires = ["setuptools>=42", "wheel", "setuptools_scm"]
+# deprecated-modules =
+
+# Output a graph (.gv or any supported image format) of external dependencies to
+# the given file (report RP0402 must not be disabled).
+# ext-import-graph =
+
+# Output a graph (.gv or any supported image format) of all (i.e. internal and
+# external) dependencies to the given file (report RP0402 must not be disabled).
+# import-graph =
+
+# Output a graph (.gv or any supported image format) of internal dependencies to
+# the given file (report RP0402 must not be disabled).
+# int-import-graph =
+
+# Force import order to recognize a module as part of the standard compatibility
+# libraries.
+# known-standard-library =
+
+# Force import order to recognize a module as part of a third party library.
+known-third-party = ["enchant"]
+
+# Couples of modules and preferred modules, separated by a comma.
+# preferred-modules =
+
+[tool.pylint.logging]
+# The type of string formatting that logging methods do. `old` means using %
+# formatting, `new` is for `{}` formatting.
+logging-format-style = "old"
+
+# Logging modules to check that the string format arguments are in logging
+# function parameter format.
+logging-modules = ["logging"]
+
+[tool.pylint."messages control"]
+# Only show warnings with the listed confidence levels. Leave empty to show all.
+# Valid levels: HIGH, CONTROL_FLOW, INFERENCE, INFERENCE_FAILURE, UNDEFINED.
+confidence = ["HIGH", "CONTROL_FLOW", "INFERENCE", "INFERENCE_FAILURE", "UNDEFINED"]
+
+# Disable the message, report, category or checker with the given id(s). You can
+# either give multiple identifiers separated by comma (,) or put this option
+# multiple times (only on the command line, not in the configuration file where
+# it should appear only once). You can also use "--disable=all" to disable
+# everything first and then re-enable specific checks. For example, if you want
+# to run only the similarities checker, you can use "--disable=all
+# --enable=similarities". If you want to run only the classes checker, but have
+# no Warning level messages displayed, use "--disable=all --enable=classes
+# --disable=W".
+disable = ["raw-checker-failed", "bad-inline-option", "locally-disabled", "file-ignored", "suppressed-message", "useless-suppression", "deprecated-pragma", "use-symbolic-message-instead", "missing-function-docstring", "missing-class-docstring", "missing-module-docstring", "unused-argument", "invalid-name", "consider-using-f-string", "too-few-public-methods", "attribute-defined-outside-init", "access-member-before-definition", "protected-access", "dangerous-default-value", "unused-variable", "no-member", "duplicate-code", "too-many-statements", "too-many-locals", "too-many-branches", "too-many-ancestors", "too-many-nested-blocks", "global-statement", "too-many-lines", "too-many-instance-attributes", "useless-option-value", "unknown-option-value", "too-many-return-statements", "line-too-long", "fixme", "cyclic-import", "unused-import", "too-many-arguments"]
```

FYI, this is where the global pylint ignores are now. Although, these should probably delegated to separate lines:
```suggestion
disable = [
    "raw-checker-failed",
    "bad-inline-option",
    "locally-disabled",
    "file-ignored",
    "suppressed-message",
    "useless-suppression",
    "deprecated-pragma",
    "use-symbolic-message-instead",
    "missing-function-docstring",
    "missing-class-docstring",
    "missing-module-docstring",
    "unused-argument",
    "invalid-name",
    "consider-using-f-string",
    "too-few-public-methods",
    "attribute-defined-outside-init",
    "access-member-before-definition",
    "protected-access",
    "dangerous-default-value",
    "unused-variable",
    "no-member",
    "duplicate-code",
    "too-many-statements",
    "too-many-locals",
    "too-many-branches",
    "too-many-ancestors",
    "too-many-nested-blocks",
    "global-statement",
    "too-many-lines",
    "too-many-instance-attributes",
    "useless-option-value",
    "unknown-option-value",
    "too-many-return-statements",
    "line-too-long",
    "fixme",
    "cyclic-import",
    "unused-import",
    "too-many-arguments"
]
```